.changeset/tall-buckets-change.md

```diff
@@ -0,0 +1,282 @@
+---
+"@salt-ds/theme": minor
+---
+
+**_Theming and CSS updates_** For teams customizing theme, the following tokens will be impacted. Update them as per suggestion.
+
+### Characteristics
+
+- New tokens added to `text` characteristics
+
+```diff
++ --salt-text-notation-fontFamily: var(--salt-typography-fontFamily);
++ --salt-text-notation-fontWeight-small: var(--salt-typography-fontWeight-regular);
++ --salt-text-notation-fontWeight-strong: var(--salt-typography-fontWeight-bold);
+```
+
+- Deprecated tokens in characteristics, use replacement tokens as listed. The change is motivated by several factors including:
+  - reduced number of tokens
+  - easier theming experience
+  - `text.css` updated to `content.css` as tokens are used on more than just texts i.e. icon
+  - consolidating tokens into `text.css` to ensure all text-related tokens are within the same file, aside from text colors, they sit within `content.css`
+
+```diff
```

Comment:
Is it worth splitting this diff into multiple? With potential headings related to the groups affected so it's easier to track how far through you are

Reply:
good idea, i'll add headers for the groups